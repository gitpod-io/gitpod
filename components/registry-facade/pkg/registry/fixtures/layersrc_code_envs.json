{
    "sourceRef": "eu.gcr.io/gitpod-core-dev/build/ide/code:cw-harmonise-regfac-env.1",
    "content": {
        "b8f8ac573b826d9f52d8892e95cc9f4b8a90cc687bfe09e29b7b9727099e85dc": "eyJzY2hlbWFWZXJzaW9uIjoyLCJjb25maWciOnsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5jb250YWluZXIuaW1hZ2UudjEranNvbiIsImRpZ2VzdCI6InNoYTI1NjpmNWYwZWFjNTVhODJhMGI5YTBmZGZjZTY2YTQ0NjQ5ZDk1MDM4ZmU4Y2QzYTIxZjMxMDYxNTUyYTFkNDAxYzE2Iiwic2l6ZSI6MzEzMH0sImxheWVycyI6W3sibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjoxZGI2MzU4MjZmYmE1OTMwMjc0M2U2M2FlYTE3NDZhNGI2NzIzNzc4Y2Y0YjcwMzQxODg0Yzg4ZmViODNmYjMxIiwic2l6ZSI6MTI5MDAwNjR9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjphMmU3MzY0NTcxMTM2ZDRlZWM3MmJjMGMzMWNmNzcxYmU3MTE4OTY4Y2U0ZDdjNGViMDZhZDA0NjEyNGUyNWY5Iiwic2l6ZSI6MjUxNTM1MTJ9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjplNmQzNWU3ZTIwNDc0MWQ2NWMxMmUzZTAzMDg5ODliZTFiY2IwNjRjN2NiZDdkYjAxY2FhYjU3ZjI0MzE2YWNlIiwic2l6ZSI6MjMyNjMyNTl9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjozMjhlZmQ2ODRhNDMxYTEwMTBhMjk1MDkxMDIwOTlhNTUyMTgyMjNlN2QyNDgyOGNlMDY1YzFjZjk3ZDcyNDZkIiwic2l6ZSI6MTIwM30seyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmltYWdlLnJvb3Rmcy5kaWZmLnRhci5nemlwIiwiZGlnZXN0Ijoic2hhMjU2OjdhNGYwYjM1ZTg4ZmQyOTExYmExZGFlZGUxMzZhNDcyN2RlMzYyYzE5MGE4MTNjMDEwZGZmN2FmYzE0MmU0ODAiLCJzaXplIjoyNzB9XX0=",
        "eu.gcr.io/gitpod-core-dev/build/ide/code:cw-harmonise-regfac-env.1": "eyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmRpc3RyaWJ1dGlvbi5tYW5pZmVzdC52Mitqc29uIiwiZGlnZXN0Ijoic2hhMjU2OmI4ZjhhYzU3M2I4MjZkOWY1MmQ4ODkyZTk1Y2M5ZjRiOGE5MGNjNjg3YmZlMDllMjliN2I5NzI3MDk5ZTg1ZGMiLCJzaXplIjoxMzY4fQ==",
        "f5f0eac55a82a0b9a0fdfce66a44649d95038fe8cd3a21f31061552a1d401c16": "eyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDo0ODo1Mi43MzAwMDM4NjhaIiwiYXJjaGl0ZWN0dXJlIjoiYW1kNjQiLCJvcyI6ImxpbnV4IiwiY29uZmlnIjp7IkVudiI6WyJQQVRIPS91c3IvbG9jYWwvc2JpbjovdXNyL2xvY2FsL2JpbjovdXNyL3NiaW46L3Vzci9iaW46L3NiaW46L2JpbiIsIkdJVFBPRF9FTlZfQVBQRU5EX1BBVEg9L2lkZS9iaW46IiwiR0lUUE9EX0VOVl9TRVRfRURJVE9SPWNvZGUiLCJHSVRQT0RfRU5WX1NFVF9WSVNVQUw9Y29kZSIsIkdJVFBPRF9FTlZfU0VUX0dJVF9FRElUT1I9Y29kZSAtLXdhaXQiXX0sInJvb3RmcyI6eyJ0eXBlIjoibGF5ZXJzIiwiZGlmZl9pZHMiOlsic2hhMjU2OmM1OTEyNjM4NWIzNDA0YmY5M2NiNjJkMDMwZWFiNzQwODlmZjkxYmY2NDAzYTU1ZmFhYTUzOTVjODVlNTU4ODQiLCJzaGEyNTY6YTU5MTEyODc1OTgyZDg4MzkxMzAwZmZlMGM3NmRkNTYzMjc5ZWQwOGU0NDgxNTJhMmRkYjUyZWQ4YjQxOGRhOCIsInNoYTI1NjpjY2M5ODk3OWUyYzE3ZTBkNTc3MTNmY2IwOGY3MWZjMGU1NGMzMTNmOWIwZTUzMWY2ZjBjNjhjODFmNmExYTQ5Iiwic2hhMjU2Ojg0Njk1OTYyZjYxZjJlYjVhODdmMzI5YjYwNTlmNmUyNzYyOTQzY2MwODZlNTE3NzgzNWE5NGFhM2NhYWY1OGUiLCJzaGEyNTY6MTk3ZmQ0NGUyNTViMTMxMzNlZmRiOTgwZjI2YzcxODlkMzQ1MzIwYjNhMDMxY2E2NmUxNTIzMmVkNDU5Mjc1NSJdfSwiaGlzdG9yeSI6W3siY3JlYXRlZCI6IjIwMjEtMDItMDJUMTA6NDg6NDYuNTIyMzY5MjMzWiIsImNyZWF0ZWRfYnkiOiIvYmluL3NoIC1jICMobm9wKSBDT1BZIGRpcjphODMzZWE5ZTc1NjkzMTM2OGNlNDE1NDI0ZGIzNGUzYjk5OTkzZjY5MjE4ODI1M2ZlZGFhYWQwMDg1MWU0MTkwIGluIC9pZGUvICJ9LHsiY3JlYXRlZCI6IjIwMjEtMDItMDJUMTA6NDg6NDguMzMxODE4ODg4WiIsImNyZWF0ZWRfYnkiOiIvYmluL3NoIC1jICMobm9wKSBDT1BZIGRpcjpjNWIyMDhlYmViYjI0NjIzYTRmYmRhZjg0NGYyNThiNmM5N2ExMmU4ZGNjMTZmZWMzMTFlY2FlOGRiYzlkMTAyIGluIC9pZGUvICJ9LHsiY3JlYXRlZCI6IjIwMjEtMDItMDJUMTA6NDg6NTAuOTkxOTYyMjE4WiIsImNyZWF0ZWRfYnkiOiIvYmluL3NoIC1jICMobm9wKSBDT1BZIGRpcjo1ODAxNmRlOWE0MjEwYmE4NjMzZDJkYmEwMzMxY2IyZDIwN2VlYjM3OGU4MmM4ZDY1YWYwNDRmOGQ0NWEwMjE3IGluIC9pZGUvbm9kZSAifSx7ImNyZWF0ZWQiOiIyMDIxLTAyLTAyVDEwOjQ4OjUxLjcwMTI5OTQ5NloiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgQ09QWSBtdWx0aTo2NjI4NjMwMDc0Yzg0OGMwM2ExZWJiNTIyZDNiZjM2MTE1YmRkZWEyZTgwYTE0ZmEyOGU3MDVjZTRhZTdmNTgxIGluIC9pZGUvICJ9LHsiY3JlYXRlZCI6IjIwMjEtMDItMDJUMTA6NDg6NTEuODI5MDYwODcyWiIsImNyZWF0ZWRfYnkiOiIvYmluL3NoIC1jICMobm9wKSBDT1BZIGRpcjpmODg5MjNkNDQxMmYyYjhkODM4MzEzNGNhZjI4YjRiNzY1ZjAxYTcyYjEzMzliNTM0MjVhZmYzZWQ2NzU4YjhmIGluIC9pZGUvYmluICJ9LHsiY3JlYXRlZCI6IjIwMjEtMDItMDJUMTA6NDg6NTIuMDIxODE3MDhaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0VOVl9BUFBFTkRfUEFUSD0vaWRlL2JpbjoiLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDo0ODo1Mi4yNTgzMTAwMzZaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0VOVl9TRVRfRURJVE9SPWNvZGUiLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDo0ODo1Mi40OTY5NjIzMTVaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0VOVl9TRVRfVklTVUFMPWNvZGUiLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDo0ODo1Mi43MzAwMDM4NjhaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0VOVl9TRVRfR0lUX0VESVRPUj1jb2RlIC0td2FpdCIsImVtcHR5X2xheWVyIjp0cnVlfV19"
    },
    "envs": null
}