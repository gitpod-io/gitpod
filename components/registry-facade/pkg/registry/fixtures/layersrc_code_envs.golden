{
    "error": "",
    "layer": [
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:1db635826fba59302743e63aea1746a4b6723778cf4b70341884c88feb83fb31",
                "size": 12900064
            },
            "DiffID": "sha256:c59126385b3404bf93cb62d030eab74089ff91bf6403a55faaa5395c85e55884"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:a2e7364571136d4eec72bc0c31cf771be7118968ce4d7c4eb06ad046124e25f9",
                "size": 25153512
            },
            "DiffID": "sha256:a59112875982d88391300ffe0c76dd563279ed08e448152a2ddb52ed8b418da8"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:e6d35e7e204741d65c12e3e0308989be1bcb064c7cbd7db01caab57f24316ace",
                "size": 23263259
            },
            "DiffID": "sha256:ccc98979e2c17e0d57713fcb08f71fc0e54c313f9b0e531f6f0c68c81f6a1a49"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:328efd684a431a1010a29509102099a55218223e7d24828ce065c1cf97d7246d",
                "size": 1203
            },
            "DiffID": "sha256:84695962f61f2eb5a87f329b6059f6e2762943cc086e5177835a94aa3caaf58e"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:7a4f0b35e88fd2911ba1daede136a4727de362c190a813c010dff7afc142e480",
                "size": 270
            },
            "DiffID": "sha256:197fd44e255b13133efdb980f26c7189d345320b3a031ca66e15232ed4592755"
        }
    ],
    "envs": [
        "PATH=/ide/bin:",
        "EDITOR=code",
        "VISUAL=code",
        "GIT_EDITOR=code --wait"
    ]
}